File: consecutivescroller/src/main/java/com/thatcbin/consecutivescroller/LayoutParamsUtils.java
package com.thatcbin.consecutivescroller;

import android.view.ViewGroup;

public class LayoutParamsUtils {

    /**
     * 使子view的topMargin和bottomMargin属性无效
     *
     * @param params
     */
    public static void invalidTopAndBottomMargin(ViewGroup.MarginLayoutParams params){
        if (params != null) {
            params.topMargin = 0;
            params.bottomMargin = 0;
        }
    }
}

File: consecutivescrollerkt/src/main/java/com/thatcbin/consecutivescroller/LayoutParamsUtils.kt
package com.thatcbin.consecutivescroller

import android.view.ViewGroup

object LayoutParamsUtils {
    /**
     * 使子view的topMargin和bottomMargin属性无效
     *
     * @param params
     */
    @JvmStatic
    fun invalidTopAndBottomMargin(params: ViewGroup.MarginLayoutParams?) {
        if (params != null) {
            params.topMargin = 0
            params.bottomMargin = 0
        }
    }
}